vuu-ui/packages/vuu-popups/src/notifications/NotificationsProvider.tsx

```diff
@@ -0,0 +1,121 @@
+import React, { useState, useContext, useCallback, useEffect } from "react";
+import classNames from "classnames"
+import { getUniqueId } from "@finos/vuu-utils";
+
+import "./notifications.css"
+
+// animation times in milliseconds
+const toastDisplayDuration = 6000;
+const exitAnimationDuration = 1000;
+const verticalTransitionTime = 300
+
+const toastHeight = 56;
+const toastWidth = 300;
+const rightGap = 50;
+const topGap = 10;
+const leftGap = 10;
```

Comment:
Might be nice to include a comment similar to the one above for animation times. Specifically including units (pixels?).

Reply:
Thanks Peter :) I have added some comments for these. Also changed these to have more descriptive names and renamed some of the "gaps" to "padding" as that describes better how we are using them